helper/useragent/useragent.go

```diff
@@ -27,3 +28,15 @@ func String() string {
 	return fmt.Sprintf("Nomad/%s (+%s; %s)",
 		versionFunc(), projectURL, rt)
 }
+
+// HeaderSetter is anything that implements SetHeaders(http.Header).
+type HeaderSetter interface {
+	SetHeaders(http.Header)
+}
+
+// SetHeaders configures the User-Agent http.Header for the client.
+func SetHeaders(client HeaderSetter) {
+	client.SetHeaders(http.Header{
+		"User-Agent": []string{String()},
```

Comment:
Hum...maybe I'm missing something, but the user-agent used to be [`hashicorp/nomad`](https://github.com/hashicorp/nomad/pull/15745/files#diff-61b0f81ca4505d23bb2adf7025d2c36c4e02232a2f13ac39dce779617c954416L155) and now it's going to be empty?

Reply:
Ops..I'm still warming up my brain from the time off 😄